inlong-dataproxy/conf/dataproxy-pulsar.conf

```diff
@@ -42,12 +42,12 @@ agent1.sources.tcp-source.msg-factory-name = org.apache.inlong.dataproxy.source.
 agent1.sources.tcp-source.host = 0.0.0.0
 agent1.sources.tcp-source.port = 46801
 agent1.sources.tcp-source.max-msg-length = 524288
-agent1.sources.tcp-source.topic = test1
+#agent1.sources.tcp-source.topic = test1
```

Comment:
Why not remove this line?

Reply:
it is optional config,  and this can be as example.